### PR TITLE
[prometheus-postgres-exporter] allow set DATA_SOURCE_URI_FILE

### PR DESCRIPTION
**Pull request by mike2194**
#### What this PR does / why we need it:
allow for defining DATA_SOURCE_URI_FILE, most helpful when using vault-agent-injector

#### Which issue this PR fixes
  - fixes #1309 

#### Checklist
- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Comment by zanhsieh** [comment]
@mike2194 
Could you sign DCO please?

**Comment by mike2194** [comment]
> @mike2194
> Could you sign DCO please?

my fault, did not signoff on version bump commit.  FIXED

**Review by zanhsieh — APPROVED** [review]

File: charts/prometheus-postgres-exporter/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "0.9.0"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 2.3.5
+version: 2.3.6
```
Comment (zanhsieh) [inline charts/prometheus-postgres-exporter/Chart.yaml:5]: Hey @mike2194 

Probably needs another version bump.

```
...
"prometheus-community" has been added to your repositories
Linting chart 'prometheus-postgres-exporter => (version: "2.3.6", path: "charts/prometheus-postgres-exporter")'
Checking chart 'prometheus-postgres-exporter => (version: "2.3.6", path: "charts/prometheus-postgres-exporter")' for a version bump...
Error: Error linting charts: Error processing charts
Old chart version: 2.3.6
New chart version: 2.3.6
------------------------------------------------------------------------------------------------------------------------
 ✖︎ prometheus-postgres-exporter => (version: "2.3.6", path: "charts/prometheus-postgres-exporter") > Chart version not ok. Needs a version bump!
------------------------------------------------------------------------------------------------------------------------
Error linting charts: Error processing charts
Error: Process completed with exit code 1.
```

**Review by zanhsieh — CHANGES_REQUESTED** [review]

File: charts/prometheus-postgres-exporter/Chart.yaml (hunk 1 above)
Comment (mike2194) [reply charts/prometheus-postgres-exporter/Chart.yaml]: Done!

**Review by zanhsieh — APPROVED** [review]

**Comment by zanhsieh** [comment]
@mike2194 
Need to bump the chart version in Charts.yaml to latest. Without it, the lint-test won't pass.

**Review by monotek — CHANGES_REQUESTED** [review]
Please update chart versiobn too

**Review by monotek — CHANGES_REQUESTED** [review]
Please raise chart version

**Pull request closed without merging**